index.d.ts

```diff
@@ -1,3 +1,22 @@
+export interface Options {
+	/**
+	Prevent extracted numbers regeneration before all possibilities are exhausted.
+
+	@default false
+
+	@example
+	```
+	import uniqueRandom from 'unique-random';
+
+	const random = uniqueRandom(1, 3, {exhaustive: true});
+
+	console.log(random(), random(), random(), "<exhausted>", random(), random(), random());
```

Comment:
Having a string in the middle there is just confusing and does not help explain how it works.